<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Datos de entrada</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Datos de salida</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Opciones</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>19</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonMeters2Centimeters">
       <property name="text">
        <string>Metors a centimetors</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonCelcius2Farenheit">
       <property name="text">
        <string>Celcius a Farenheit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEditInput">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutput">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
